bin/get-ffmpeg

```diff
@@ -14,8 +14,8 @@ if [ ! -f bin/ffprobe ]; then
   mkdir -p bin
   mv ffmpeg-*-static/ffprobe bin/
   rm -rf ffmpeg-*-static
-  if [ ! -f bin/ffprobe ]; then
-    echo "ERROR: something went wrong - ffprobe not present"
+  if [[ ! -f bin/ffprobe ]]; then
+    echo "ERROR: something went wrong - ffmmpeg not present"
```

Comment:
checking for ffprobe, but error is ffmpeg?

Reply:
(and `ffmpeg` is misspelled)

Reply:
Changed the message to `ffprobe`.  This script is pretty much copied from the dovetail-stitch-lambda, and i didn't want to change it too much yet.  (And this repo will likely need `ffmpeg` too in the near future).